docs/pages/using-nhs-notify/personalisation.md

```diff
@@ -18,47 +18,37 @@ To personalise a message, use double brackets to add a placeholder to your conte
     text='<code>Hello ((firstName)), your NHS Number is ((nhsNumber))'
 %}
 
-### Personal Demographics Service (PDS) personalisation fields
+## Personal Demographics Service (PDS) personalisation fields
 
-NHS Notify uses the [Personal Demographics Service (PDS)](https://digital.nhs.uk/services/personal-demographics-service) to find and populate certain personalisation fields. You only need to [provide us with recipients' NHS numbers]({% link pages/using-nhs-notify/tell-us-who-you-want-to-message.md %}).
+NHS Notify uses the [Personal Demographics Service (PDS)](https://digital.nhs.uk/services/personal-demographics-service) to find and populate certain personalisation fields for each recipient. This happens automatically when you [tell us who you want to message]({% link pages/using-nhs-notify/tell-us-who-you-want-to-message.md %}) using recipients' NHS numbers.
 
-The personalisation fields available in PDS are:
+You can use the following PDS personalisation fields in your templates:
 
 - fullName
 - firstName
 - middleNames
 - lastName
 - namePrefix
 - nameSuffix
-- address_line_1
-- address_line_2
-- address_line_3
-- address_line_4
-- address_line_5
-- address_line_6
-- address_line_7
 - nhsNumber
 - date
-- clientRef
-- recipientContactValue
-- template
 
-The placeholders in your content need to match the PDS personalisation fields.
+Any placeholders in your content need to match the PDS personalisation fields.
 
 ## Providing your own personalisation data
 
 You can provide your own personalisation data.
 
 Do not send us personalisation data that's already available in PDS. We'll use the PDS data over your own data in this case.
 
-### If you're using the API
+### If you're using NHS Notify API
 
 Include a personalisation block in your API request.
 
 For example, if you wanted to include 'practice' as a personalisation field, the personalisation block would be:
 
 {% include components/inset-text.html
-    text='{
+text='{
```

Comment:
I'd suggest using line break tags to format this example. I.e:

`text='<code>{<br /><br />"practice": "PRACTICE_NAME",<br /><br />}</code>'`

Reply:
Raised due to weirdness with tags:
```
<div class="nhsuk-inset-text">
  <span class="nhsuk-u-visually-hidden">Information:</span>
  <p>{</p>

<div class="language-plaintext highlighter-rouge"><div class="highlight"><pre class="highlight"><code>"practice": "PRACTICE_NAME",
</code></pre></div></div>

<p>}</p>

</div>
```

Resolved by removing `<code></code>` tags